### PR TITLE
Hotfix/refactor observation

### PR DESCRIPTION
**Pull request by wesleymonte**

### CODE REVIEW

File: probes/src/main/java/cloud/fogbow/probes/core/fta/FtaSender.java [hunk 1]
```diff
@@ -22,6 +23,7 @@ public static void sendObservation(String address, Observation observation) {
         try {
             LOGGER.info("Sending observation to [" + address + "]");
             StringEntity body = toJson(observation);
+            LOGGER.debug("Observation json body: " + EntityUtils.toString(body));
```
Comment (joivo) [inline probes/src/main/java/cloud/fogbow/probes/core/fta/FtaSender.java]: Remove this log.

File: probes/src/main/java/cloud/fogbow/probes/core/models/Value.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
+package cloud.fogbow.probes.core.models;
+
+import org.json.JSONObject;
+
+public class Value {
```
Comment (joivo) [inline probes/src/main/java/cloud/fogbow/probes/core/models/Value.java:9]: What represents a value?

**Review by joivo — APPROVED** [review]
Just a bit change.

File: probes/src/main/java/cloud/fogbow/probes/core/fta/FtaSender.java (hunk 1 above)
Comment (wesleymonte) [reply probes/src/main/java/cloud/fogbow/probes/core/fta/FtaSender.java]: Done.

File: probes/src/main/java/cloud/fogbow/probes/core/models/Value.java (hunk 2 above)
Comment (wesleymonte) [reply probes/src/main/java/cloud/fogbow/probes/core/models/Value.java:9]: Added doc for explanation.

**Review by joivo — APPROVED** [review]

**Pull request merged**